src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/SendDailyReportTask.java

```diff
@@ -0,0 +1,97 @@
+package uk.gov.hmcts.reform.blobrouter.tasks;
+
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.blobrouter.model.out.EnvelopeSummaryItem;
+import uk.gov.hmcts.reform.blobrouter.services.email.EmailSender;
+import uk.gov.hmcts.reform.blobrouter.services.report.ReportCsvWriter;
+import uk.gov.hmcts.reform.blobrouter.services.report.ReportService;
+
+import java.io.File;
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+@Component
+@ConditionalOnProperty("scheduling.task.send-daily-report.enabled")
+public class SendDailyReportTask {
+    private static final Logger logger = LoggerFactory.getLogger(SendDailyReportTask.class);
+
+    static final String EMAIL_SUBJECT = "Reform Scan daily report";
+    static final String EMAIL_BODY = "This is an auto generated email. Do not respond to it.";
```

Comment:
I can see that this message is taken from bulk-scan-processor, but we should go for `autogenerated` or `auto-generated`, as it's incorrect as two words.

Reply:
Will do